src/components/SkillRatingCard/SkillRatingCard.js

```diff
@@ -1,5 +1,9 @@
 // Packages
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Cookies from 'universal-cookie';
```

Comment:
Move Cookies under `// Components` section and correct the spelling of Components.